### ISSUE TITLE
Insertions lack the expected inserted sequence in the ALT field

### ISSUE THREAD

**Issue by biorococo** [fritzsedlazeck/Sniffles#608]
Hi,

I noticed that some insertion variants do not contain the expected inserted sequence in the ALT field of the output VCF. Instead, the ALT allele is reported as \<INS\>.

See couple of example records:

```groovy

chr1	17289303	Sniffles2.INS.216S0	T	<INS>	60	PASS	PRECISE;SVTYPE=INS;SVLEN=5256;END=17289303;SUPPORT=15;COVERAGE=33,42,42,32,33;STRAND=+;NM=0.005;PHASE=2,17007696,1,1,PASS,PASS;STDEV_LEN=0;STDEV_POS=0;SUPPORT_LONG=14;VAF=0.833	GT:GQ:DR:DV:PS	1|1:27:7:35:17007696
chr2	241596525	Sniffles2.INS.870S1	G	<INS>	60	PASS	PRECISE;SVTYPE=INS;SVLEN=82;END=241596525;SUPPORT=8;COVERAGE=21,29,29,27,22;STRAND=+-;NM=0.016;PHASE=2,241579962,7,7,FAIL,PASS;STDEV_LEN=1.915;STDEV_POS=0.500;SUPPORT_LONG=0;VAF=0.276	GT:GQ:DR:DV:PS	0/1:21:21:8:241579962
``` 

Could you clarify under what conditions Sniffles reports the insertion sequence in the ALT field versus using the symbolic <INS> allele?

Let me know if you need any other information.

Thanks a lot in advance.

**Comment by fritzsedlazeck** [comment]
This is expected every now and then. Should be more often in larger insertions. This is when the module cannot come to a consensus for the insertions either due to size or other mapping difficulties.
Thanks 
Fritz
